### PR TITLE
Add Puppet status report

### PR DESCRIPTION
**Pull request by smortex**

### CODE REVIEW

File: website/content/en/status/report-2023-04-2023-06/puppet.adoc [hunk 1]
```diff
@@ -0,0 +1,16 @@
+=== Puppet
+
+Links: +
+link:https://puppet.com/docs/puppet/latest/puppet_index.html[Puppet] URL: link:https://puppet.com/docs/puppet/latest/puppet_index.html[] +
+
+Contact: Puppet Team <puppet@FreeBSD.org>
+
+Puppet is a Free Software configuration management tool, composed of a source of trust (Puppet Server) that describes the expected configuration of machines with a domain-specific language, and an agent (Puppet Agent) on each node which enforce the actual configuration match the expected one. An optional database (PuppetDB) can be setup for reporting and describing advanced schemas where the configuration of a machine depends on the configuration of another one.
```
Comment (lsalvadore) [inline website/content/en/status/report-2023-04-2023-06/puppet.adoc:8]: s/which enforce the actual configuration match the expected one/which enforces that the actual configuration matches the expected one/

File: website/content/en/status/report-2023-04-2023-06/puppet.adoc [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+Since our last status report, the Puppet Team maintained the Puppet 6 and Puppet 7 ports up to date.
```
Comment (lsalvadore) [inline website/content/en/status/report-2023-04-2023-06/puppet.adoc:10]: I suggest something more direct and more focused on today: "The Puppet team is maintaining Puppet 6 and Puppet 7 ports, which are all up to date."

File: website/content/en/status/report-2023-04-2023-06/puppet.adoc [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+Puppet 8 has been recently released and was added to the ports tree.
```
Comment (lsalvadore) [inline website/content/en/status/report-2023-04-2023-06/puppet.adoc:12]: s/was added/has been added/

File: website/content/en/status/report-2023-04-2023-06/puppet.adoc [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+Puppet 6 has reached End of Life and has been deprecated.  It is now expired.  Users of Puppet 6 are therefore advised to update to Puppet 7 or Puppet 8.
+
+For now, the Puppet 7 remains the default Puppet version for ports depending on Puppet.  The Puppet Community is hard at work making sure the various Puppet modules work with the latest code and at the time of writing this report, updating to Puppet 8 may be challenging.  The situation is getting better every day, and we expect to switch Puppet 8 as the default version of Puppet in a few months, when the wave of modules update is finished.
```
Comment (lsalvadore) [inline website/content/en/status/report-2023-04-2023-06/puppet.adoc:16]: s/switch Puppet 8/switch to Puppet 8/

**Review by lsalvadore — CHANGES_REQUESTED** [review]
Thanks for your report, I made a few inline comments.

Could you also please put one sentence per line? Thanks.

File: website/content/en/status/report-2023-04-2023-06/puppet.adoc (hunk 4 above)
Comment (igalic) [inline website/content/en/status/report-2023-04-2023-06/puppet.adoc:16]: s/For now, the Puppet 7 remains the default/For now, Puppet 7 remains the default/

Puppet version for ports depending on Puppet.  The Puppet Community is hard at work making sure the various Puppet modules work with the latest code and at the time of writing this report, updating to Puppet 8 may be challenging.

s/The situation is getting better every day, and we expect to switch Puppet 8 as the default/The situation is getting better every day, and we expect to switch to Puppet 8 as the default/

**Review by igalic — CHANGES_REQUESTED** [review]
aside from the suggested changes, i highly recommend writing one sentence per line, as that makes these reviews easier, as well as future diffs.

although status reports don't usually see much editing after release, it's a very good idea to get used to this style

**Comment by smortex** [comment]
@lsalvadore, @igalic I amended my commit to include your changes and separate each sentence with a new line. Thanks!

File: website/content/en/status/report-2023-04-2023-06/puppet.adoc [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+Puppet 8 has been recently released and has been added to the ports tree.
+
+Puppet 6 has reached End of Life and has been deprecated.
+It is now expired.
+Users of Puppet 6 are therefore advised to update to Puppet 7 or Puppet 8.
+
+For now, Puppet 7 remains the default Puppet version for ports depending on Puppet.
+The Puppet Community is hard at work making sure the various Puppet modules work with the latest code and at the time of writing this report, updating to Puppet 8 may be challenging.
+The situation is getting better every day, and we expect to switch to Puppet 8 as the default version of Puppet in a few months, when the wave of modules update is finished.
```
Comment (pauamma) [inline website/content/en/status/report-2023-04-2023-06/puppet.adoc:21]: s/modules update/module updates/

**Review by pauamma — APPROVED** [review]

**Review by lsalvadore — APPROVED** [review]

**Pull request closed without merging**